test/module/shared_model/bindings/ClientTest.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
+
+import java.math.BigInteger;
+import java.io.InputStream;
+import java.io.IOException;
+
+import iroha.protocol.*;
+import iroha.protocol.BlockOuterClass.Transaction;
+import iroha.protocol.Commands.Command;
+import com.google.protobuf.ByteString;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ClientTest {
+  static {
+    try {
+      System.loadLibrary("irohajava");
+    } catch (UnsatisfiedLinkError e) {
+      System.err.println("Native code library failed to load. \n" + e);
+      System.exit(1);
+    }
+  }
+
+  private Keypair keys;
+
+  @BeforeEach
+  void setUp() {
+    keys = new ModelCrypto().generateKeypair();
+  }
+
+  Command.Builder validAddPeer() {
+    return Command.newBuilder().setAddPeer(Commands.AddPeer.newBuilder().setPeer(
+        Primitive.Peer.newBuilder()
+            .setAddress("127.0.0.1:50500")
+            .setPeerKey(ByteString.copyFrom(String.format("%32.32s", " ").getBytes()))));
+  }
+
+  Transaction.Payload.ReducedPayload.Builder defaultPayload() {
+    return Transaction.Payload.ReducedPayload.newBuilder()
+        .setCreatorAccountId("admin@test")
+        .setCreatedTime(System.currentTimeMillis())
```

Comment:
you should fix time

Reply:
by "fix" I mean, you should hardcode the value

Reply:
Exactly. Bindings do not allow to fix values in texts. Talk about this with team